### PR TITLE
Modify ComponentExampleBox component (#1647)

### LINKED ISSUE #1647: Modify ComponentExampleBox component
This component needs some modifications.
Here is the list of issues that should be improved:
- The UI of the component header buttons should be more interactive.
- While displaying the component preview, the box should not have a vertical scroll.
- We should add the `@code {...}` keyword in the razor file of the component so that it would be displayed in all the samples.
- The URL provided for each sample doesn't work correctly in the live env.

### PR DESCRIPTION
**Pull request by maryamhdr**
this closes #1647

### CODE REVIEW

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/ComponentExampleBox.razor [hunk 1]
```diff
@@ -29,15 +30,17 @@
                        IconName="BitIconName.Copy"
                        Title="Copy source">
             </BitIconButton>
-            
+
             <pre class="code">
-                <code class="html">@HTMLSourceCode</code>
+            <code class="html">@HTMLSourceCode</code>
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/ComponentExampleBox.razor:35]: code formatting issue

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Styles/_colors.scss [hunk 2]
```diff
@@ -1,21 +1,4 @@
...
-$DarkGreen: #064006;
-$DarkOrange: #D4783F;
-$DarkYellow: #E0CC12;
-$LightBlue: #63A5DB;
-$LightGreen: #2E9952;
-$LightPurple: #CB42F5;
-//=============================================================
-
-$B1Color: #0054C6;
+﻿$B1Color: #0054C6;
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Styles/_colors.scss:1]: What's going on here? are these colors redundant?

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/ComponentExampleBox.razor (hunk 1 above)
Comment (maryamhdr) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/ComponentExampleBox.razor:35]: The code formatting is ok...If we apply the formatting here, the result will not be what we expect.

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Styles/_colors.scss (hunk 2 above)
Comment (maryamhdr) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Styles/_colors.scss:1]: These colors are redundant and I had removed them before.
But I think the changes have been missed in a merge conflict.

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/ComponentExampleBox.razor (hunk 1 above)
Comment (msynk) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/ComponentExampleBox.razor:35]: I see, ok

**Review by msynk — APPROVED** [review]

**Pull request merged**